File: src/core/RunningProblem.java
package src.core;

public class RunningProblem extends Problem<String, Integer> {

    public static void main(String[] args) {
        new RunningProblem().run();
    }

    @Override
    protected TestCase<String, Integer>[] getTestCases() {
        return new TestCase[]{};
    }

    @Override
    protected Integer solve(String testCaseInput) {
        return null;
    }
}

File: src/core/RunningProblem.kt
package src.core

class RunningProblem : Problem<Int, Boolean>() {

    companion object {
        @JvmStatic
        fun main(args: Array<String>) = RunningProblem().run()
    }

    override fun getTestCases(): Array<TestCase<Int, Boolean>> = arrayOf()

    override fun solve(testCaseInput: Int): Boolean {
        return false
    }
}
